core/src/main/java/com/opensymphony/xwork2/ognl/accessor/XWorkListPropertyAccessor.java

```diff
@@ -36,6 +36,7 @@
  * this class will create the necessary blank JavaBeans.
  *
  * @author Gabriel Zimmerman
+ * @author Yanming Zhou <zhouyanming@gmail.com>
```

Comment:
Please remove author tag. In general you [should refrain from doing so](http://struts.apache.org/releases.html#clarifications). Why? [Because](http://directory.apache.org/apacheds/coding-standards.html#classinterface-headers).